Package.swift

```diff
@@ -20,5 +20,8 @@ let package = Package(
     targets: [
         .target(name: "Swinject",
                 path: "Sources"),
+        .testTarget(name: "SwinjectTests",
+                    dependencies: ["Swinject"],
+                    path: "Tests/SwinjectTests"),
```

Comment:
```suggestion
```
Seems redundant since it's already on the default path.

Reply:
Regardless, a blessed PR

Reply:
Ah thanks for pointing that out, updated the PR to remove the custom path